validator-network/src/network_impl.rs

```diff
@@ -178,60 +196,18 @@ where
         validator_id: usize,
         msg: M,
```

Comment:
`M` likely no longer needs to be `Clone`. Can't mention it there, but it is 3 lines up.

Reply:
Done.